src/@next/Switch/SwitchStyle.tsx

```diff
@@ -54,9 +76,25 @@ export const InputStyle = styled.input`
     }
 
     &:before {
-      transform: translate(16px, -50%);
+      ${({ withIcon }: SwitchProps) => ({
```

Comment:
here as well, we can rely on `data-with-icon` attribute

Reply:
done